install.sh

```diff
@@ -27,6 +35,18 @@ RETRY_DELAY=0
 INSECURE=${INSECURE:=}
 EXTRA_CURL_PARAMS=""
 EXTRA_WGET_PARAMS=""
+
+log "DISABLE_DOCKER_CHECKS = $DISABLE_DOCKER_CHECKS"
+log "DISABLE_REPORTING = $DISABLE_REPORTING"
+log "DISABLE_CLUSTER_CREATION = $DISABLE_CLUSTER_CREATION"
+log "FILES_BRANCH = $FILES_BRANCH"
+log "DATA_VOLUME = $DATA_VOLUME"
+log "INSTALL_ID = $INSTALL_ID"
+log "USE_LOCAL_HELM = $USE_LOCAL_HELM"
+log "USE_GPU = $USE_GPU"
+log "INSECURE = $INSECURE"
```

Comment:
All the `logs` call are going to fail (and fail the script) the first time you run the script because the directory won't exist

Reply:
ok lets add a mkdir then?